Week 0/Reverse String/reverse_string.java

```diff
@@ -0,0 +1,16 @@
+
+// https://www.geeksforgeeks.org/collections-swap-method-in-java-with-examples/
```

Comment:
Esse link se refere ao uso da função swap em Java. Essa função não está sendo usada no código, então deixar esse link aí talvez confunda alguém que for ler. 

Em vez desse link, seria melhor ter o link do desafio do LeetCode referente a esse problema 🤖